database/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -602,7 +602,7 @@ class SlickDatabase(databaseConfig: Config) extends SqlDatabase {
     runTransaction(action)
   }
 
-  protected def addMetadata(events: Iterable[Metadatum])
+  override def addMetadata(events: Iterable[Metadatum])
```

Comment:
To be clear, `Metadatum` is _not_ a "Slick type"-- it is a database object.

The exclusive list of database types currently lives under [`cromwell.database.obj`](https://github.com/broadinstitute/cromwell/tree/adbae9e33267f83afd6203f1a223bc6e59d24d42/database/src/main/scala/cromwell/database/obj).

If one ever wanted to replace slick, there would still be a database object for each table, and within each database object the fields would still map to the database columns one-for-one.